tdrs-frontend/src/actions/reports.test.js

```diff
@@ -1,7 +1,28 @@
 import thunk from 'redux-thunk'
 import configureStore from 'redux-mock-store'
 
-import { setYear, SET_YEAR } from './reports'
+import { upload, SET_FILE, setYear, SET_YEAR } from './reports'
+
+describe('actions/reports.js', () => {
+  // Init a mock store to test redux actions
+  const mockStore = configureStore([thunk])
+
+  it('should dispatch SET_FILE', async () => {
+    const store = mockStore()
+
+    await store.dispatch(
+      upload({ file: { name: 'HELLO' }, section: 'Active Case Data' })
+    )
+
+    const actions = store.getActions()
+
+    expect(actions[0].type).toBe(SET_FILE)
+    expect(actions[0].payload).toStrictEqual({
+      fileName: 'HELLO',
```

Comment:
is there a reason the filename is 'hello'?

Reply:
No; this is an arbitrary name in these redux action tests.

Reply:
Note that this set of tests don't touch UI components so no file validation happens.